uportal-war/src/main/java/org/jasig/portal/portlets/dynamicskin/FileSystemDynamicSkinService.java

```diff
@@ -47,6 +31,23 @@
 import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.stereotype.Service;
 
+import javax.portlet.PortletContext;
+import javax.portlet.PortletPreferences;
+import javax.portlet.PortletRequest;
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.FilenameFilter;
+import java.io.IOException;
+import java.net.URL;
+import java.text.MessageFormat;
+import java.util.Enumeration;
+import java.util.Iterator;
+import java.util.Map;
+import java.util.Set;
+import java.util.SortedSet;
+import java.util.TreeSet;
+import java.util.concurrent.CopyOnWriteArraySet;
```

Comment:
FWIW it would be good to update your intelliJ to use the same order as eclipse which is the uPortal standard to avoid this noise; e.g. see 

![intellijimportsettings](https://cloud.githubusercontent.com/assets/1479823/13560690/7b66fcb2-e3df-11e5-90fe-3c3f18262b28.png)